app/controllers/books_controller.rb

```diff
@@ -1,12 +1,12 @@
 class BooksController < ApplicationController
   before_action :find_book, only: :show
   def index
-    if params[:search].blank?
-      @books = Book.page(params[:page]).per Settings.page_book
-    else
-      @books = Book.search(params[:search].downcase)
-        .page(params[:page]).per Settings.page_book
-    end
+    @books = if params[:search].blank?
```

Comment:
dùng toán tử 3 ngôi đi e

Reply:
@manhbnt-0960 dài này 3 ngôi ntn được :v

Reply:
@books = params[:search].blank? ? Book : Book.search(params[:search].downcase)
@books = @books.page(params[:page]).per Settings.page_book